SCF-Assignment-8/Bird.java

```diff
@@ -0,0 +1,23 @@
+package Question1;
+
+/**
+ * Bird is subclass of Animal.
+ * 
+ * @author Arjita
+ *
+ */
+public class Bird extends Animal {
+	public Bird(String name, int age, String sound) {
+		super(name, age, sound);

```

Comment:
We can pass the Category type from constructor only, hence we won't be required using switch case for checking the Animal Category in the Animal class.